python/tvm/topi/cuda/nms.py

```diff
@@ -610,8 +610,10 @@ def _get_sorted_indices(data, data_buf, score_index, score_shape):
     )
 
     target = tvm.target.Target.current()
-    # TODO(masahi): Check -libs=thrust option
-    if target and target.kind.name in ["cuda", "rocm"] and is_thrust_available():
+    if target and (
+        can_use_thrust(target, "tvm.contrib.thrust.sort")
+        or can_use_rocthrust(target, "tvm.contrib.thrust.sort")
+    ):
         sort_tensor = argsort_thrust(score_tensor, axis=1, is_ascend=False, dtype="int32")
     else:
         sort_tensor = argsort(score_tensor, axis=1, is_ascend=False, dtype="int32")
```

Comment:
Just a thought and feel free to ignore: Since THRUST won't be used without `-libs=thrust` after this PR, would that be better to add a warning message here if `tvm.contrib.thrust.sort` is available (i.e., users built TVM with THRUST enabled) and target doesn't have `-libs=thrust`? We can then remove the warning in the next release or so.

Reply:
Sure I'll do that after I get more feedback

Reply:
Warning added in https://github.com/apache/tvm/pull/7468/commits/607f2d591a36f14f5013090bb843da44554cb8c6

Please have a look